style/README.md

```diff
@@ -346,3 +346,12 @@ Testing
   appear in the class.
 
 [Imperative mood]: http://en.wikipedia.org/wiki/Imperative_mood
+
+Legacy Code
```

Comment:
I agree with @thegreatape. Is there a case where we should be using `deprecate` now or in the past?